export.py

```diff
@@ -147,16 +148,28 @@ def export_onnx(model, im, file, opset, dynamic, simplify, prefix=colorstr('ONNX
         elif isinstance(model, DetectionModel):
             dynamic['output0'] = {0: 'batch', 1: 'anchors'}  # shape(1,25200,85)
 
-    torch.onnx.export(
-        model.cpu() if dynamic else model,  # --dynamic only compatible with cpu
-        im.cpu() if dynamic else im,
-        f,
-        verbose=False,
-        opset_version=opset,
-        do_constant_folding=True,
-        input_names=['images'],
-        output_names=output_names,
-        dynamic_axes=dynamic or None)
+    if sparsified:
```

Comment:
this is not relevant to this PR at all but i'd be nice if we could just add our stuff as an additional pass after a torch.onnx.export call. ModuleExporter already calls onnx.export under the hood, so it'd be cool if we could do this as
```
torch.onnx.export(...)
if sparsified:
   sparseml.simplify_onnx(f)
```